webapp/scripts/functions.js

```diff
@@ -269,3 +270,16 @@ function formatDesc(rawArray) {
   }
   return table
 }
+
+function round(val, n) {
+  let i = 0
+  let dot = false
+  while (val[i] && ['0', '.'].indexOf(val[i]) > -1) {
+    if (val[i] === '.') dot = true
+    i++
+  }
+  if (!dot) {
+    while (val[i] && val[i] != '.') i++
+  }
+  return val.substring(0, i + n + 1)
+}
```

Comment:
What does this function do?

Reply:
for example: it rounds 1231.13351451 to 1231.13 and doesn't make change to integers

Reply:
Have you considered https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toPrecision ?

Reply:
the Number.prototype.toPrecision() rounds any number to the precision given. E.g. Number.parseFloat(12121).toPrecision(4) would give "1.212e+4"... but I guess we want "12121" instead.

Reply:
Hmm, probably, yes.